.github/workflows/main-build.yml

```
@@ -25,7 +25,7 @@ jobs:
   testpatch:
     name: Test patches applied to dependencies
     runs-on: aws-otel-java-instrumentation_ubuntu-latest_32-core
-    if: ${{ startsWith(github.ref_name, 'release/v') }}
+    if: ${{ hashFiles('.github/patches/**/opentelemetry-java-*.patch') != ''}}
```

```suggestion
    if: ${{ hashFiles('.github/patches/**/opentelemetry-java-*.patch') != ''}}
```
I don't think we need this conditional anymore. Running the patch action is a NOOP if no patch files exist.